assets/src/application/ui/layout/entityList/filterBar/buttons/ViewSwitchBtnGroup.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+
+import { ButtonGroup } from '@application/ui/input';
+import CardViewFilterButton from './CardViewFilterButton';
+import TableViewFilterButton from './TableViewFilterButton';
+import { ViewSwitchBtnGroupProps } from '../types';
+
+import { getPropsAreEqual } from '@appServices/utilities';
+
+const ViewSwitchBtnGroup: React.FC<ViewSwitchBtnGroupProps> = ({ listId, setCardView, setTableView, view }) => {
```

Comment:
Not sure if the name `ViewSwitchBtnGroup` is clear enough. My be it should be something like `EntityViewButtons`? Our CNO will have a better suggestion :)

Reply:
agree, I've picked `ViewSwitchBtnGroup` as the first and simplest option that came to mind, but def needs CNO's superpowers.

Reply:
- ToggleViewButtonGroup
- ListViewButtonGroup
- EntityListViewButtonGroup

last one might be the most clear

Reply:
renamed to the most clear option. thx